src/components/PeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -189,40 +206,69 @@ export default class PeopleAndRoles extends Vue {
   }
 
   /**
-   * Sets properties to add an org/person.
+   * Determines whether we have the specified role by count and mode.
```

Comment:
Did we want the params described here? I'm not really sure what the `mode` is for example.

Reply:
Done.